src/app/components/BibPage/BibDetails.jsx

```diff
@@ -14,144 +14,166 @@ import Actions from '../../actions/Actions';
 import DefinitionList from './DefinitionList';
 import appConfig from '../../../../appConfig.js';
 
-const getIdentifiers = (bibValues, fieldIdentifier) => {
-  let val = '';
-
-  if (bibValues.length) {
-    bibValues.forEach(value => {
-      if (value.indexOf(`${fieldIdentifier}:`) !== -1) {
-        val = value.substring(fieldIdentifier.length + 1);
-      }
-    });
+class BibDetails extends React.Component {
```

Comment:
If those functions are used several places, we can put them in another js as a module.